lib/loader/index.js

```diff
@@ -14,9 +14,13 @@
  */
 
 var global = require('../global');
+var namespacer = require('../namespacer');
 var doc = global.document;
 
 var DEFAULT_TIMEOUT = 10000;
+var SHARED_NAMESPACE_NAME = 'BV_LOADER_TRACKING';
```

Comment:
How would you feel about defaulting to SHARED_NAMESPACE_NAME, but allow users the option of passing in a specific namespace? Best of both worlds?